tests/phpunit/tests/multisite/wpMsUsersListTable.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+if ( is_multisite() ) :
+
+	/**
+	 * @group admin
+	 * @group network-admin
+	 *
+	 * @covers WP_MS_Users_List_Table
+	 */
+	class Tests_Multisite_wpMsUsersListTable extends WP_UnitTestCase {
+		protected static $site_ids;
+
+		/**
+		 * @var WP_MS_Users_List_Table
+		 */
+		public $table = false;
+
+		public function set_up() {
+			parent::set_up();
+			$this->table = _get_list_table( 'WP_MS_Users_List_Table', array( 'screen' => 'ms-users' ) );
+		}
+
+		public static function wpSetUpBeforeClass( WP_UnitTest_Factory $factory ) {
+			self::$site_ids = array(
+				'wordpress.org/'          => array(
+					'domain' => 'wordpress.org',
+					'path'   => '/',
+				),
+				'wordpress.org/foo/'      => array(
+					'domain' => 'wordpress.org',
+					'path'   => '/foo/',
+				),
+				'wordpress.org/foo/bar/'  => array(
+					'domain' => 'wordpress.org',
+					'path'   => '/foo/bar/',
+				),
+				'wordpress.org/afoo/'     => array(
+					'domain' => 'wordpress.org',
+					'path'   => '/afoo/',
+				),
+				'make.wordpress.org/'     => array(
+					'domain' => 'make.wordpress.org',
+					'path'   => '/',
+				),
+				'make.wordpress.org/foo/' => array(
+					'domain' => 'make.wordpress.org',
+					'path'   => '/foo/',
+				),
+				'www.w.org/'              => array(
+					'domain' => 'www.w.org',
+					'path'   => '/',
+				),
+				'www.w.org/foo/'          => array(
+					'domain' => 'www.w.org',
+					'path'   => '/foo/',
+				),
+				'www.w.org/foo/bar/'      => array(
+					'domain' => 'www.w.org',
+					'path'   => '/foo/bar/',
+				),
+				'test.example.org/'       => array(
+					'domain' => 'test.example.org',
+					'path'   => '/',
+				),
+				'test2.example.org/'      => array(
+					'domain' => 'test2.example.org',
+					'path'   => '/',
+				),
+				'test3.example.org/zig/'  => array(
+					'domain' => 'test3.example.org',
+					'path'   => '/zig/',
+				),
+				'atest.example.org/'      => array(
+					'domain' => 'atest.example.org',
+					'path'   => '/',
+				),
+			);
+
+			foreach ( self::$site_ids as &$id ) {
+				$id = $factory->blog->create( $id );
+			}
+			unset( $id );
+		}
+
+		public static function wpTearDownAfterClass() {
+			foreach ( self::$site_ids as $site_id ) {
+				wp_delete_site( $site_id );
+			}
+		}
+
+		/**
+		 * @ticket 42066
+		 *
+		 * @covers WP_MS_Users_List_Table::get_views
+		 */
+		public function test_get_views_should_return_views_by_default() {
+			$expected = array(
+				'all'   => '<a href="http://example.org/wp-admin/network/users.php" class="current" aria-current="page">All <span class="count">(1)</span></a>',
+				'super' => '<a href="http://example.org/wp-admin/network/users.php?role=super">Super Admin <span class="count">(1)</span></a>',
```

Comment:
These counts are based on a local install. The test is faulty and just needs to use `get_user_count()` and `count( get_super_admin() )` instead. I'll push this fix along with commits to fix docblock alignment and the other failing tests.

Reply:
Fixed in [be0780b](https://github.com/WordPress/wordpress-develop/pull/2828/commits/be0780b54a7325231e8baea27de462fac6ecb213).